src/service_router.rs

```diff
@@ -104,71 +104,77 @@ pub fn create<T: Controller>(controller: T) -> Chain {
     chain
 }
 
-
-// FIXME: Declare this crate in the test module, and avoid the use of pub
-// When this comment was written, putting these 2 lines in before_each returns a compile error:
-// ``` error: unresolved import `self::iron_test::request`. Could not find `iron_test` in
-// service_router::service_router` [E0432] ```
-extern crate iron_test;
```

Comment:
Nice! Thanks for finding how to fix it. 

For reference, @ferjm gave me the trick on IRC: you have to do the import in main.rs